ts/components/conversation/AddMentions.tsx

```diff
@@ -31,10 +31,14 @@ class Mention extends React.Component<MentionProps, MentionState> {
   }
 
   public componentWillMount() {
-    const found = this.findMember(this.props.text.slice(1));
-    this.setState({ found });
 
-    this.tryRenameMention();
+    this.setState({ found: false });
+
+    this.findMember(this.props.text.slice(1)).then((found) => {
+      this.setState({ found });
```

Comment:
tryRenameMention already sets state, if found. Why do it again?

Reply:
Right... actually, how about I just call `tryRenameMention` here? (see updated code)